applications/crossbar/src/modules/cb_sms.erl

```diff
@@ -253,7 +255,7 @@ create_sms_doc_id() ->
 -spec summary(cb_context:context()) -> cb_context:context().
 summary(Context) ->
     {View, PreFilter, PostFilter} = get_view_and_filter(Context),
```

Comment:
Just trying to understand your PR in full - why did you include these API changes in this PR?  How are they related to the other changes?